src/Raven.Studio/typescript/widgets/virtualGrid/columns/providers/documentBasedColumnsProvider.ts

```diff
@@ -193,7 +193,7 @@ class documentBasedColumnsProvider {
         const docDto = doc.toDto(true);
         
         const text = JSON.stringify(docDto, null, 4);
-        const title = doc.getId() ? "Document: " + doc.getId() : "Document preview";
+        const title = doc.getId() ? "Document: " + doc.getId() : "Index Entry preview";
```

Comment:
we are in document based column provider. Assuming that document w/o id is index entry might be misleading. Can you provide optional parameter to this function to be title provider? 
```
static showPreview(doc: document, customTitle?: string) {
        const docDto = doc.toDto(true);
        
        const text = JSON.stringify(docDto, null, 4);
        const title = customTitle ?? (doc.getId() ? "Document: " + doc.getId() : "Document preview");
        app.showBootstrapDialog(new showDataDialog(title, text, "javascript"));
    }
```

Reply:
or even stringly index entry and call `app.showBootstrapDialog(new showDataDialog("Index Entry preview", text, "javascript"));`  from your code